### PR TITLE
Renamed Audiologist Log In to just Log In on several components

### PR DESCRIPTION
**Pull request by wynnliam**
* **What is the new behavior (if this is a feature change)?**
Renames buttons that have 'Audiologist Log In' to 'Log In'

### CODE REVIEW

File: client/src/app/patient-summary/patient-summary.component.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
         id="aud-button" 
         (click)="audioLoginClick()" 
         type="button" 
-        class="btn btn-primary btn-sm">Audiologist Log In</button>
+        class="btn btn-primary btn-sm">Log In</button>
```
Comment (ElijahRW) [inline client/src/app/patient-summary/patient-summary.component.html:8]: Actually, I think we should rename this button to “Submit Survey” or “Start Tests”. Anyone else have thoughts?

**Review by ElijahRW — APPROVED** [review]
Made a potential suggestion, otherwise, looks good!!!

File: client/src/app/patient-summary/patient-summary.component.html (hunk 1 above)
Comment (MishalAlajmi-psu) [reply client/src/app/patient-summary/patient-summary.component.html:8]: Start Tests sounds better than Submit Survey, and I do think that it makes more sense than Log In in that page. I think its a good idea.

**Review by MishalAlajmi-psu — APPROVED** [review]
Looks good, I would just implement Elijah's suggestion.

**Pull request merged**